torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -976,4 +987,28 @@ at::Scalar PythonArgs::scalar_slow(int i) {
   return at::Scalar(THPUtils_unpackDouble(args[i]));
 }
 
+void extract_torch_function_overloaded_arg(PyObject* arg, std::vector<py::handle>& overloaded_args) {
```

Comment:
This is a duplicate of code in PythonArgParser and it should be cross-referenced here and there accordingly

Reply:
This logic is a subset in FunctionParameter::check, https://fburl.com/diffusion/vc724gz6. I am not sure how to share the whole 'extract_torch_function_overloaded_arg' with PythonArgParser without large refactoring, since we need to match the schema ahead (we don't have a good way to match the schema yet for custom op in python), then we can try to generate a string fmt to create FunctionParameter. One thing I could do here is lift the logic of handling TENSOR_LIST to a function, so it could be shared here and there.

Reply:
okay, I have merge this logic with the logic in FunctionParameter::check.